Type error when using strict type checking

Hello! I've come across an interesting bug in the type definitions when recently updating

```
$ tsc
node_modules/@octokit/webhooks/index.d.ts:5004:17 - error TS7010: 'createWebhooksApi', which lacks return-type annotation, implicitly has an 'any' return type.

5004 export function createWebhooksApi(options?: Options);
                     ~~~~~~~~~~~~~~~~~


Found 1 error.
```

I can work around this using the `skipLibCheck` option for `tsc`, but skipping all 3rd party type defs is clearly not ideal :smile: 

Any chance you'd consider updating the types here?

--- a/index.d.ts
+++ b/index.d.ts
@@ -5001,7 +5001,7 @@ declare class Webhooks {
   ): void | Promise<void>;
 }
 
-export function createWebhooksApi(options?: Options);
+export function createWebhooksApi(options?: Options): Webhooks;
 
 export default Webhooks;
 export { Webhooks };

--- a/scripts/generate-types.js
+++ b/scripts/generate-types.js
@@ -75,7 +75,7 @@ declare class Webhooks {
   public middleware (request: http.IncomingMessage, response: http.ServerResponse, next?: (err?: any) => void): void | Promise<void>
 }
 
-export function createWebhooksApi(options?: Options);
+export function createWebhooksApi(options?: Options): Webhooks;
 
 export default Webhooks;
 export { Webhooks };

--- a/test/typescript-validate.ts
+++ b/test/typescript-validate.ts
@@ -1,4 +1,4 @@
-import * as Webhooks from "..";
+import Webhooks from "..";
 import { createServer } from "http";
 
 // ************************************************************

